velox/expression/tests/ExprTest.cpp

```diff
@@ -97,7 +97,11 @@ class ExprTest : public testing::Test, public VectorTestBase {
       const VectorPtr& base,
       vector_size_t index) {
     return std::make_shared<core::ConstantTypedExpr>(
-        std::make_shared<ConstantVector<T>>(execCtx_->pool(), 1, index, base));
+        BaseVector::wrapInConstant(1, index, base));
+  }
+
+  std::shared_ptr<core::ConstantTypedExpr> makeConstantExpr(variant value) {
```

Comment:
Does this interface work for complex types? If not, could we add a comment?

Reply:
This API supports only primitive types. Added a comment.